chain/chain/src/store.rs

```diff
@@ -2292,4 +2360,57 @@ mod tests {
         assert_ne!(block_hash, block_hash1);
         assert_ne!(epoch_id_to_hash, epoch_id_to_hash1);
     }
+
+    #[test]
```

Comment:
Can you also test that `clear_old_data` works for multiple heights?

Reply:
++